src/Symfony/Bridge/Monolog/Processor/RouteProcessor.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Monolog\Processor;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\FinishRequestEvent;
+use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+/**
+ * Adds the current route information to the log entry.
+ *
+ * @author Piotr Stankowski <git@trakos.pl>
+ */
+class RouteProcessor implements ProcessorInterface, EventSubscriberInterface
+{
+    private $routeData = array();
+    private $includeParams;
+
+    public function __construct(bool $includeParams = true)
+    {
+        $this->includeParams = $includeParams;
+    }
+
+    public function __invoke(array $records)
+    {
+        if (!empty($this->routeData) && !isset($records['extra']['requests'])) {
```

Comment:
`if ($this->routeData && !isset($records['extra']['requests'])) {`

Reply:
OK, changed.